.testgrid-database.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2019, WSO2 Inc. (http://wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# TestGrid Test Configuration Parameters
+version: '0.9'
+emailToList: "manurip@wso2.com"
```

Comment:
shall we add builder@ mail as well here

Reply:
As mentioned offline, waiting for approval for a separate mailing list.